greenmail-core/src/main/java/com/icegreen/greenmail/store/SimpleMessageAttributes.java

```diff
@@ -737,4 +736,9 @@ public static Header create(String[] header) {
             return new Header(header[0]);
         }
     }
+
+    public String escapeHeader(final String text) {
+        return text.replace("\r", "\\\r").replace("\n", "\\\n").replace("\\", "\\\\").replace("\"", "\\\"");

```

Comment:
Using MimeUtility.unfold for the CLRFs worked great, but replacing the existing quotations with MimeUtility.quote (which only optionally quotes) caused several test failures.